docker-compose-files/openshift-compose/docker-compose.yml

```diff
@@ -0,0 +1,143 @@
+version: "2"
+services:
+  hello:
+    image: surajd/hello
+    ports:
+    - "8080"
+    - "8778"
+    - "8443"
+    labels:
+     - "app=hello"
+     - "hystrix.enabled=true"
+
+  hola:
+    image: containscafeine/hola
+    ports:
+    - "8080"
+    labels:
+    - "app=hola"
+    - "hystrix.enabled=true"
+
+  aloha:
+    image: surajd/aloha
+    ports:
+    - "8080"
+    - "8778"
+    labels:
+    - "app=aloha"
+    - "hystrix.enabled=true"
+
+  ola:
+    image: surajd/ola
+    ports:
+    - "8080"
+    - "8778"
+    labels:
+    - "app=ola"
+    - "hystrix.enabled=true"
+
+  namaste:
+    image: surajd/namaste
+    ports:
+    - "8080"
+    - "8778"
+    labels:
+    - "app=namaste"
+    - "hystrix.enabled=true"
+
+  bonjour:
+    image: surajd/bonjour
+    ports:
+    - "8080"
+    labels:
+    - "app=bonjour"
+    - "hystrix.enabled=true"
+
+  api-gateway:
+    image: surajd/api-gateway
+    ports:
+    - "8080"
+    - "8778"
+    labels:
+    - "app=api-gateway"
+    - "hystrix.enabled=true"
+
+  hystrix-dashboard:
+    image: fabric8/hystrix-dashboard:1.0.15
```

Comment:
So these labels makes no sense right now, because OpenShift oc import ignores these labels, so updating the PR by removing those labels.